CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # bedrock-vue-wallet ChangeLog
 
+## 22.0.2 - 2023-10-xx
```

Comment:
Since this is considered an addition and not just a bug fix, it should be a minor version bump.

```suggestion
## 22.1.0 - 2023-10-dd
```